src/app/app.component.ts

```diff
@@ -350,14 +364,14 @@ export class AppComponent implements AfterViewInit {
     return !!this.entrySelected;
   }
 
-  onEntrySelected(event) {
+  onEntrySelected(event: any) {
     const customEvent = event as CustomEvent<LfTreeNode[]>;
```

Comment:
type event as CustomEvent<LfTreeNode[]>

Reply:
typing this as CustomEvent<LfTreeNode[]> led to type errors when we passed an object of type Event from the view. I changed the event to be of type Event instead of any, and I left the cast in the first line of the function